tfe/resource_tfe_team_access.go

```diff
@@ -44,6 +49,78 @@ func resourceTFETeamAccess() *schema.Resource {
 				),
 			},
 
+			"permissions": {
```

Comment:
this is actually kinda how i wish the api was, with the individual permissions nested under a permissions block 😄

Reply:
A discussion was had about that, actually; it was decided in the end that it's too confusing with several API responses already containing `permissions` hashes that relate to what _you are able to do with the resource_, and not 'what you are granting through this resource as here.